### PR TITLE
OCPBUGS#1823: Deprecation note update

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OCPBUGS-1823

Link to docs preview: https://53222--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-deprecated-features-crio-parameters

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -1319,7 +1319,7 @@ In {product-title} 4.11, support for VMware ESXi 7.0 Update 1 or earlier is depr
 [id="ocp-4-11-deprecated-features-crio-parameters"]
 ==== Support for `pidsLimit` and `logSizeMax` CRI-O parameters will be deprecated
 
-In {product-title} 4.11, support for the `pidsLimit` and `logSizeMax` CRI-O parameters is expected to be deprecated in a future release. It is recommended to use the `podPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
+In {product-title} 4.11, the `pidsLimit` and `logSizeMax` the `ContainerRuntimeConfig` CR will be deprecated and dropped in a future release. Use the `podPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```
Comment (aireilly) [inline release_notes/ocp-4-11-release-notes.adoc:1322]:
```suggestion
In {product-title} 4.11, the `pidsLimit` and `logSizeMax` fields in the `ContainerRuntimeConfig` CR will be deprecated and will be dropped in a future release. Use the `podPidsLimit` and `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -1319,7 +1319,7 @@ In {product-title} 4.11, support for VMware ESXi 7.0 Update 1 or earlier is depr
 [id="ocp-4-11-deprecated-features-crio-parameters"]
 ==== Support for `pidsLimit` and `logSizeMax` CRI-O parameters will be deprecated
 
-In {product-title} 4.11, support for the `pidsLimit` and `logSizeMax` CRI-O parameters is expected to be deprecated in a future release. It is recommended to use the `podPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
+In {product-title} 4.11, the `pidsLimit` and `logSizeMax` the `ContainerRuntimeConfig` CR will be deprecated and removed in a future release. Use the `podPidsLimit` and the `ContainerLogMaxSize` fields in the `KubeletConfig` CR instead. The default value of the `podPidsLimit` field is `4096`.
```
Comment (haircommander) [inline release_notes/ocp-4-11-release-notes.adoc:1322]: nit: 
```
`logSizeMax` fields in the `ContainerRuntimeConfig`...
```

Comment (Amrita42) [reply release_notes/ocp-4-11-release-notes.adoc]: looks good, I am not sure if  additional **the** is required in the below:
 
Use the `podPidsLimit` and **the** `ContainerLogMaxSize` fields in the `KubeletConfig` CR

**Comment by mburke5678** [comment]
@amolnar-rh Travis error needs to be addressed.

**Comment by amolnar-rh** [comment]
@mburke5678 I was waiting for the fix, should be good to go now :)

**Pull request merged**